templates/set-lcm-mode.ps1.erb

```diff
@@ -0,0 +1,51 @@
+Write-Host "Setting LCM"
+$LCMRefreshMode = '<%= @refresh_mode %>'
+$generationDate = (get-date -format "M/d/yyyy HH:mm:ss")
+$encoding       = [system.Text.Encoding]::UTF8
+$Namespace      = 'root/Microsoft/Windows/DesiredStateConfiguration'
+$ClassName      = 'MSFT_DSCLocalConfigurationManager'
+
+Write-Verbose "LCM Mode: $LCMRefreshMode"
+
+$mofData = @"
+/*
+@TargetNode=''localhost''
+@GeneratedBy=PUPPET
+@GenerationDate=$generationDate
+@GenerationHost=PUPPET
```

Comment:
Was wondering if we should be setting this to the target node's hostname instead of Puppet since it's technically generated on the target node and not on puppet.